lib/ccd/templates/result_observation_template.rb

```diff
@@ -2,34 +2,33 @@ module Ccd::ResultObservationTemplate
   def self.included(base)
     base.class_eval do
       extend ::Ccd::Dsl
-      
+
       # SHALL contain exactly one [1..1] @classCode="OBS" Observation (CodeSystem: HL7ActClass 2.16.840.1.113883.5.6 STATIC) (CONF:7130).
       constraint 'class_code', {:cardinality=>"1..1", :value=>"OBS"}
-      
+
       # SHALL contain exactly one [1..1] @moodCode="EVN" Event (CodeSystem: ActMood 2.16.840.1.113883.5.1001 STATIC) (CONF:7131).
       constraint 'mood_code', {:cardinality=>"1..1", :value=>"EVN"}
-      
+
       # SHALL contain exactly one [1..1] templateId (CONF:7136) such that it
-      constraint 'template_id', {:cardinality=>"1..1"}
-      
       # SHALL contain exactly one [1..1] @root="2.16.840.1.113883.10.20.22.4.2" (CONF:9138).
       constraint 'template_id.root', {:cardinality=>"1..1", :value=>"2.16.840.1.113883.10.20.22.4.2"}
```

Comment:
does this work?  I've been converting this line to 

```suggestion
      constraint 'template_id', {:cardinality=>"1..1", :value=>{:root=>"2.16.840.1.113883.10.20.22.4.2"}}
```

Reply:
I assume so since that's what what the gem had in there but I'll do what you've been doing since that's been working 👍